chinese_tutorial_rvest_httr.Rmd

```diff
@@ -0,0 +1,490 @@
+
+### 1.1 常用函数
+
+- rvest 包中常用函数一览：
+
+  | Function Name       | Meaning                                          | meaningChinese                     |
+  | ------------------- | ------------------------------------------------ | ---------------------------------- |
+  | `back`              | History navigation tools                         | 导航工具                           |
+  | `encoding`          | Guess and repair faulty character encoding.      | 猜测并修复错误的字符编码。         |
+  | `follow_link`       | Navigate to a new url.                           | 导航到一个新的url。                |
+  | `google_form`       | Make link to google form given id                | 使链接到谷歌形式给定的id           |
+  | `guess_encoding`    | Guess and repair faulty character encoding.      | 猜测并修复错误的字符编码。         |
+  | `html`              | Parse an HTML page.                              | 解析HTML页面。                     |
+  | `html_attr`         | Extract attributes, text and tag name from html. | 从html中提取属性、文本和标记名称。 |
+  | `html_attrs`        | Extract attributes, text and tag name from html. | 从html中提取属性、文本和标记名称。 |
+  | `html_children`     | Extract attributes, text and tag name from html. | 从html中提取属性、文本和标记名称。 |
+  | `html_form`         | Parse forms in a page.                           | 解析页面中的表单。                 |
+  | `html_name`         | Extract attributes, text and tag name from html. | 从html中提取属性、文本和标记名称。 |
+  | `html_node`         | Select nodes from an HTML document               | 从HTML文档中选择节点               |
+  | `html_nodes`        | Select nodes from an HTML document               | 从HTML文档中选择节点               |
+  | `html_session`      | Simulate a session in an html browser.           | 在html浏览器中模拟会话。           |
+  | `html_table`        | Parse an html table into a data frame.           | 将html表解析为数据帧。             |
+  | `html_text`         | Extract attributes, text and tag name from html. | 从html中提取属性、文本和标记名称。 |
+  | `is.session`        | Simulate a session in an html browser.           | 在html浏览器中模拟会话。           |
+  | `jump_to`           | Navigate to a new url.                           | 导航到一个新的url。                |
+  | `pluck`             | Extract elements of a list by position.          | 按位置提取列表的元素。             |
+  | `read_xml.response` | Parse an HTML page.                              | 解析HTML页面。                     |
+  | `read_xml.session`  | Parse an HTML page.                              | 解析HTML页面。                     |
+  | `repair_encoding`   | Guess and repair faulty character encoding.      | 猜测并修复错误的字符编码。         |
+  | `session_history`   | History navigation tools                         | 历史记录导航工具                   |
+  | `set_values`        | Set values in a form.                            | 在表单中设置值。                   |
+  | `submit_form`       | Submit a form back to the server.                | 将表单提交回服务器。               |
+
+### 1.2 节点定位方法
```

Comment:
Hashtag headers should not be followed by numbers since the hashtags will create numbered subheadings

Reply:
This problem has been solved, thank you.